java/hive/src/main/java/io/v6d/hadoop/fs/FileSystem.java

```diff
@@ -378,18 +380,54 @@ public boolean renameInternal(Path path, Path path1) throws IOException {
             Files.delete(nioFilePath);
         } else {
             Files.move(nioFilePath, nioFilePath1);
+            ByteBuffer bytes = ByteBuffer.allocate(255);
+            FileChannel channel = FileChannel.open(nioFilePath1, StandardOpenOption.READ);
+            int len = channel.read(bytes);
+            String objectIDStr =
+                    new String(bytes.array(), 0, len, StandardCharsets.UTF_8).replaceAll("\n", "");
+            IPCClient client = Context.getClient();
+            client.putName(ObjectID.fromString(objectIDStr), nioFilePath1.toString());
```

Comment:
No `xxx`, `xxx1`. I remember that I have fixed them and using `src`/`dst` instead. Why there are still those meaningless identifiers?

Reply:
Sorry. I fixed it.